cvat-ui/src/components/create-job-page/create-job-page.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import './styles.scss';
+
+import React, { useEffect, useState } from 'react';
+import { Row, Col } from 'antd/lib/grid';
+import { useParams } from 'react-router';
+import Text from 'antd/lib/typography/Text';
+import Spin from 'antd/lib/spin';
+import { Task } from 'reducers';
+import { notification } from 'antd';
+import { getCore } from 'cvat-core-wrapper';
+import JobForm from './job-form';
+
+const core = getCore();
+
+function CreateJobPage(): JSX.Element {
+    const [fetchingTask, setFetchingTask] = useState(true);
+    const [taskInstance, setTaskInstance] = useState<Task | null>(null);
+
+    const id = +useParams<{ id: string }>().id;
+    useEffect((): void => {
+        if (Number.isInteger(id)) {
+            core.tasks.get({ id })
+                .then(([task]: Task[]) => {
+                    if (task) {
+                        setTaskInstance(task);
+                    }
+                }).catch((error: Error) => {
+                    notification.error({
+                        message: 'Could not fetch requested task from the server',
+                        description: error.toString(),
+                    });
+                }).finally(() => {
+                    setFetchingTask(false);
+                });
```

Comment:
After any async requests in hooks need to be sure that the component is still mounted before updating its state, otherwise it leades to errors to the browser console (changing state of unmounted component). 

You can use custom hook for that:
```
import { useIsMounted } from '@root/utils/hooks';
const isMounted = useIsMounted()
if (isMounted()) { ...  updateState .... }
```

Reply:
Please, check the same in other functional components